torch/_inductor/codecache.py

```diff
@@ -1900,7 +1949,12 @@ def format_consts_to_cpp(
                 BuildOption=object_build_options,
             )
             consts_o = object_builder.get_target_file_path()
-            object_builder.build()
+            if use_asm_build is False and is_zero_size_consts:
+                src = normalize_path_separator(str(consts_s))
+                asm_cwd = normalize_path_separator(str(consts_s.parent))
```

Comment:
Better to call `normalize_path_separator` inside of RunAsmBuildObject

Reply:
updated